azuredevops/internal/acceptancetests/resource_wiki_page_test.go

```diff
@@ -0,0 +1,95 @@
+//go:build (all || wiki || resource_wiki) && !exclude_resource_wiki
+// +build all wiki resource_wiki
+// +build !exclude_resource_wiki
+
+package acceptancetests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+)
+
+func TestAccWikiPageResource_Basic(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+
+	tf := "azuredevops_wiki.project_wiki"
+	resourceType := "azuredevops_wiki"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: checkWikiDestroyed(resourceType),
+		Steps: []resource.TestStep{
+			{
+				Config: hclProjectWikiPage(projectName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("azuredevops_wiki.project_wiki", "project_id"),
+					resource.TestCheckResourceAttrSet("azuredevops_wiki.project_wiki", "type"),
+					resource.TestCheckResourceAttrSet("azuredevops_wiki.project_wiki", "name"),
+					resource.TestCheckResourceAttrSet("azuredevops_wiki_page.wiki_page", "project_id"),
+					resource.TestCheckResourceAttrSet("azuredevops_wiki_page.wiki_page", "wiki_id"),
+					resource.TestCheckResourceAttrSet("azuredevops_wiki_page.wiki_page", "path"),
+					resource.TestCheckResourceAttrSet("azuredevops_wiki_page.wiki_page", "content"),
+				),
+			},
+			{
+				ResourceName:      tf,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccWikiPageResource_CreateAndUpdate(t *testing.T) {
```

Comment:
Import test case should be added since this resource support import.
Ref: https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/acceptancetests/resource_agent_pool_test.go#L88

Reply:
hi @xuzhang3 

Failed to import a basic example as [wiki](https://github.com/microsoft/terraform-provider-azuredevops/blob/main/website/docs/r/wiki.html.markdown#import).
```tf
resource "azuredevops_wiki_page" "eee" {
  project_id = azuredevops_project.project.id
  wiki_id = azuredevops_wiki.project_wiki.id
  path = "/eee"
  content = "eee"
}

```
```bash
$ terraform import azuredevops_wiki_page.test '1988'
azuredevops_wiki_page.test: Importing from ID "1988"...
azuredevops_wiki_page.test: Import prepared!
  Prepared azuredevops_wiki_page for import
azuredevops_wiki_page.eee: Refreshing state... [id=1988]
2024-09-26T22:15:20.420+0800 [ERROR] vertex "import azuredevops_wiki_page.eee result" error: Argument args.Project can not be nil or empty
2024-09-26T22:15:20.420+0800 [ERROR] vertex "azuredevops_wiki_page.eee (import id \"1988\")" error: Argument args.Project can not be nil or empty
2024-09-26T22:15:20.420+0800 [ERROR] vertex "azuredevops_wiki_page.eee (expand)" error: Argument args.Project can not be nil or empty
│ Error: Argument args.Project can not be nil or empty
```

Considering about that wiki page contains project_id, wiki_id and page_id itself.
There are some example might related to this kind of property. Should I try this first?
https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/service/feed/resource_feed.go#L31

https://github.com/microsoft/terraform-provider-azuredevops/blob/main/website/docs/r/feed.html.markdown#import